ui/apps/dashboard/src/components/Billing/Addons/EntitlementListItemSelfService.tsx

```diff
@@ -111,15 +111,7 @@ export default function EntitlementListItemSelfService({
               if this problem persists.
             </p>
           )}
-          {entitlement.displayValue && !openSelfService && (
-            <p
-              className={`text-basis pr-3 text-sm ${
-                typeof entitlement.displayValue === 'string' ? 'font-medium' : ''
-              }`}
-            >
-              {entitlement.displayValue}
-            </p>
-          )}
+          {entitlement.displayValue && !openSelfService && entitlement.displayValue}
```

Comment:
Something like this would make the code more readable: 

{entitlement.displayValue && !openSelfService ? entitlement.displayValue : null}